tellerlogger/src/main/java/com/kiliansteenman/teller/logger/ui/detail/TellerDetailActivity.kt

```diff
@@ -57,20 +64,16 @@ internal class TellerDetailActivity : AppCompatActivity(R.layout.teller_detail)
 
     private fun onStateChanged(state: TellerDetailViewState) {
         when (state) {
-            is TellerDetailViewState.Success -> {
-                onLogLoaded(state.tellerLog)
-            }
-            else -> {
-                // Do nothing
-            }
+            is TellerDetailViewState.Success -> onLogLoaded(state.tellerLog)
+            else -> Unit // Do nothing
         }
     }
 
     private fun onLogLoaded(log: TellerLog) {
         findViewById<TextView>(R.id.detail_title).text = log.title
         findViewById<TextView>(R.id.detail_timestamp).text = log.logDate.formatTimeStamp()
         findViewById<TextView>(R.id.detail_type).text = log.type
-        findViewById<TextView>(R.id.detail_content).text = log.content
+        tellerAdapter.setParameters(log.params)
```

Comment:
Unrelated: Looks at all these `findViewById()` calls 🙈 

(you can ignore this)

Reply:
Yeah it's terrible... I miss compose